lib/sidekiq/instrument/middleware/server.rb

```diff
@@ -6,11 +6,18 @@ class ServerMiddleware
 
     def call(worker, job, queue, &block)
       StatsD.increment(metric_name(worker, 'dequeue'))
+      DogStatsD.increment('sidekiq.dequeue', worker_dog_options(worker))
 
-      StatsD.measure(metric_name(worker,'runtime'), &block)
+      start_time = Time.now
+      yield block
+      execution_time_ms = (Time.now - start_time) * 1000
```

Comment:
I don't think you need to compute in ms, the gems will handle converting to correct units. 

```suggestion
      execution_time_ms = (Time.now - start_time)
```

Reply:
based on our conversations, the * 1000 is correct, as [timing](https://www.rubydoc.info/github/DataDog/dogstatsd-ruby/master/Datadog/Statsd#timing-instance_method) method takes in millisecond, and the diff is always in seconds